dashboard/status/tests/fixtures.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+from os import path
+from json import load
```

Comment:
I think for ``json`` also will be more readable to use ``import json``, not ``from json import load``